README.md

```diff
@@ -37,3 +37,31 @@ The relayer consists of the following components:
 <div align="center">
   <img src="resources/relayer-diagram.png?raw=true">
 </div>
+
+## Testing
+
+---
+
+### Unit tests
+
+Unit tests can be ran locally by running the command in root of the project:
+
+```bash
+./scripts/test.sh
+```
+
+### E2E tests
+
+E2E tests are ran as part of CI, but can also be ran locally. To run the E2E tests locally, you need to have the `avalanchego` build path and vm binary set up. For example with [subnet-evm](https://github.com/ava-labs/subnet-evm):
```

Comment:
Can you note that the you need clone subnet-evm locatlly and check out the correct version? Maybe we could leverage `versions.sh` to automate this.